### DISCUSSION TITLE
dc_obs_climo in zppy compy

### DISCUSSION THREAD

**Discussion by araman2** [E3SM-Project/zppy#368] [General]
I'm using zppy v1.5.1 in compy. Could you please point me to the equivalent path for dc_obs_climo in compy? 
dc_obs_climo = '/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/obs/climatologydc_obs_climo = '/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/obs/climatology

**Comment by forsyth2** [comment]
https://github.com/E3SM-Project/zppy/blob/main/tests/integration/utils.py is a good resource for finding corresponding paths. 

For Compy, we have `"diags_obs_climo": f"{diags_base_path}/observations/Atm/climatology/",`.

You can run the following to get `diags_base_path`:
```
$ python -c 'import mache; print(mache.MachineInfo().config.get("diagnostics", "base_path"))'
/compyfs/diagnostics
```
Or, if you don't have `mache` installed, you can go to https://github.com/E3SM-Project/mache/blob/main/mache/machines/compy.cfg directly: `base_path = /compyfs/diagnostics`

So, this tells us the corresponding path would be ` /compyfs/diagnostics/observations/Atm/climatology/`

**Comment by chengzhuzhang** [comment]
Hi @forsyth2, thank you for the reply. I was asked by a few other folks about the paths on different machines. One thing that I think will help is to add generated configuration files in the repo, in this case it is more straightforward to see all the configurations including paths for each machines..

**Reply by forsyth2** [reply]
I had considered that when I first created the generation script, but decided against it because of the user-specific paths (i.e., `scratch`, `user_output`, and `user_www`). I suppose it would be helpful though; created #369.

**Reply by forsyth2** [reply]
Example `cfg`s can now be found at https://github.com/E3SM-Project/zppy/tree/main/tests/integration/generated

**Reply by chengzhuzhang** [reply]
This is very helpful! Thank you @forsyth2 for taking the effort to include these machine-specific cfg files!
